bundle-workflow/python/sign.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import os
+import subprocess as sp
+import tempfile
+import uuid
+import argparse
+from manifests.bundle_manifest import BundleManifest
+
+parser = argparse.ArgumentParser(description = "Sign artifacts")
+parser.add_argument('manifest', type = argparse.FileType('r'), help="Manifest file.")
+args = parser.parse_args()
+
+def sign(filename):
+    signatureFile = filename + ".asc"
+    signing_cmd = './opensearch-signer-client -i ' + filename + ' -o ' + signatureFile + ' -p pgp'
+    sp.run(signing_cmd.split(),cwd='opensearch-signer-client/src')
```

Comment:
I believe this doesn't fail if the command fails, you'll need to `check_output`.